lib/PAUSE/package.pm

```diff
@@ -83,12 +83,27 @@ sub give_regdowner_perms {
   my $self = shift;
   my $dbh = $self->connect;
   my $package = $self->{PACKAGE};
+  my $main_package = $self->{MAIN_PACKAGE};
   local($dbh->{RaiseError}) = 0;
   # warn "Going to execute [SELECT userid FROM mods WHERE modid = '$package']";
-  my ($sth_mods) = $dbh->selectrow_hashref("SELECT userid
+  my $sql = "SELECT userid
                                 FROM   mods
-                                WHERE  modid = ?", undef, $package);
-  if ($sth_mods) { # make sure we regard the owner as the owner
+                                WHERE  modid = ?
+                                ";
+  my @args = ($package);
+  unless(lc($main_package) eq lc($package)) {
+    $sql .= q{
+    UNION
+    SELECT userid
+    FROM   perms
+    WHERE  LOWER(package) = LOWER(?)
```

Comment:
Case sensitivity is controlled by collation settings, which can be set globally, at the database level and at the table level. The default *might* be case-insensitive, but one would need access to the database to be sure.